e2e/test/iothub/FileUploadE2ETests.cs

```diff
@@ -130,10 +130,7 @@ private async Task UploadFileGranularAsync(Stream source, string filename, Http1
                 cert = s_selfSignedCertificate;
                 x509Auth = new DeviceAuthenticationWithX509Certificate(testDevice.Id, cert);
 
-                // The X509 certificate being used for device authentication needs to be set into FileUploadTransportSettings as well,
-                // so that the HttpClient created for file upload operation has access to those certificates.
-                clientOptions.FileUploadTransportSettings.ClientCertificate = cert;
-                deviceClient = DeviceClient.Create(testDevice.IoTHubHostName, x509Auth, Client.TransportType.Http1, clientOptions);
```

Comment:
this was updated in master

Reply:
This is for the new Fairfax support. Is it directly merged to preview?

Reply:
this is sequence followed here:

master -> previews/pnpApi
update previews/pnpApi
previews/pnpApi -> preview

Reply:
We had this additional x509 certificate initialization for file upload operations, as a part of our e2e tests. This is not required and se the test was updated in master.